apps/shipments/serializers.py

```diff
@@ -203,8 +205,16 @@ def update(self, instance, validated_data):
 
     def validate_device_id(self, device_id):
         auth = self.context['auth']
+        shipment = self.context['shipment']
+        if not device_id or device_id == 'null':
```

Comment:
We don't want people passing in the string 'null' as a device id, they should be passing in an empty string.

Reply:
I had to have it as either null or None, as if you enter it in as a raw JSON it does not allow a blank string. That immediately fails, as the device is set as blank=False